### DISCUSSION TITLE
ETH & ETC uncle blokcs are orphaned

### DISCUSSION THREAD

**Discussion by yq0810** [coinfoundry/miningcore#1598] [General]
From the perspective of the integrity of the program code, conitnue is missing when confirming the uncle block, which leads to the wrong entry of if block.ConfirmationProgress > 0.75 , cause uncle blocks to become orphan blocks

New pull request, try to fix it
https://github.com/oliverw/miningcore/pull/1574

**Comment by david739** [comment]
hello, you fixed it, it keeps writing to me at the miner (consecutive shares rejected by the pool)

**Comment by blackmennewstyle** [comment]
Hello,

I think you still have a typo in your code, you declare `var matchUncle = false;` at line `161` and `217`.

At line `226` you have:
```
if (matchUncle)
                    continue;
```
There is no case when `matchUncle = true;` in your code :smiling_face_with_tear: so how can it work?

**Comment by blackmennewstyle** [comment]
There were actually a fix already for the issue: https://github.com/oliverw/miningcore/pull/1501#issue-1439302693

You just need to follow what he suggested in his comment, increase the value for the option `BlockSearchOffset`in your `config.json` file, it works great.

**Comment by oliverw** [comment]
> There were actually a fix already for the issue: [#1501 (comment)](https://github.com/oliverw/miningcore/pull/1501#issue-1439302693)
> 
> You just need to follow what he suggested in his comment, increase the value for the option `BlockSearchOffset`in your `config.json` file, it works great.

Interesting 💡 !
